.coderabbit.yaml

```
@@ -0,0 +1,106 @@
+  - "**/*.yml"
+
+# Review settings
+review:
+  # Focus on code quality and best practices
+  focus:
+    - "code-quality"
+    - "security"
+    - "performance"
+    - "maintainability"
+    - "test-coverage"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)